src/osd/osd_types.h

```diff
@@ -2236,9 +2258,11 @@ struct pg_stat_t {
   int32_t acting_primary;
 
   // snaptrimq.size() is 64bit, but let's be serious - anything over 50k is
-  // absurd already, so cap it to 2^32 and save 4 bytes at  the same time
+  // absurd already, so cap it to 2^32 and save 4 bytes at the same time
```

Comment:
could you keep the cleanup code here and in other places, in a separate commit, given that we need to backport this all the way back